src/Learning/KWDataPreparation/KWClassStats.cpp

```diff
@@ -1177,28 +1197,29 @@ void KWClassStats::WriteJSONFields(JSONFile* fJSON)
 	// Parametrage eventuel de l'apprentissage supervise
 	if (GetTargetAttributeName() != "")
 	{
+		assert(not GetLearningSpec()->IsTargetStatsComputed() or
```

Comment:
Add a comment above, someting like (to keep some explanation, after the removal of the previous comment):
```c++
On accepte meme on affichage incomplet au cas ou le traitement aurait echoue partiellement ou totalement
```

Reply:
Fait